drivers/starburst/src/metabase/driver/implementation/query_processor.clj

```diff
@@ -221,3 +237,40 @@
   [_ _ expr]
   (let [report-zone (qp.timezone/report-timezone-id-if-supported :starburst)]
     (hsql/call :from_unixtime expr (hx/literal (or report-zone "UTC")))))
+
+(defmethod sql.qp/->honeysql [:starburst :datetime-diff]
```

Comment:
Do you know if metabase has provided tests for this? Now that it is enabled, lets say you got this logic wrong, are there new tests in the suite of tests that would fail? 

We should confirm that. If metabase didn't supply tests for this, we likely need to add one to our test suite, but usually they do add tests for these features.

Reply:
https://github.com/metabase/metabase/blob/0975f5d9b2daf5737ad65fec8be4c3d4c65287dd/test/metabase/query_processor_test/date_time_zone_functions_test.clj#L699